components/brave_wallet_ui/components/extension/confirm-transaction-panel/zcash-transaction-info.tsx

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Lets rename this file to use `snake_case` convention.

`zcash_transaction_info.tsx`